doc/Example-Gradle-Project/build.gradle

```diff
@@ -0,0 +1,146 @@
+// This file is part of JavaSMT,
+// an API wrapper for a collection of SMT solvers:
+// https://github.com/sosy-lab/java-smt
+//
+// SPDX-FileCopyrightText: 2021 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Apache-2.0
+
+apply plugin: 'java'
+apply plugin: 'application'
+
+mainClassName = 'org.sosy_lab.java_smt_example.SolverOverviewTable'
+
+repositories {
+  // try to resolve by pom first
+  mavenCentral {
+    metadataSources {
+      mavenPom()
+    }
+  }
+    // try to get artifacts if no useful pom is found
+  mavenCentral {
+    metadataSources {
+      artifact()
+    }
+  }
+    // Ivy can be used as well
+  ivy {
+    url "https://www.sosy-lab.org/ivy"
+    patternLayout {
+      artifact "/[organisation]/[module]/[classifier]-[revision].[ext]"
+      //ivy "/[organisation]/[module]/ivy-[revision].xml"
+    }
+    metadataSources {
+       artifact()
+    }
+  }
+  // SMT-Solver Princess repo
+  maven {
+    allowInsecureProtocol true
+    url "http://logicrunch.research.it.uu.se/maven/"
+  }
+}
+
+configurations {
+  // Use a custom config that inherits all dependencies from runtimeOnly and implementation.
+  // If you want to seperate your dependencies from the JavaSMT ones
+  // simply use it when pulling the dependencies.
+  javaSMTConfig.extendsFrom(runtimeOnly, implementation)
+}
+
+// You can manage the JavaSMT dependencies via a config if you need to split them from your regular dependencies
+dependencies {
+  // JUnit 4
+  testImplementation 'junit:junit:4.13'
+
+  // Retrieve JavaSMT with Maven and transitive dependencies via a pom.xml
+  implementation 'org.sosy-lab:java-smt:3.11.0'
+
+  // Retrieve Z3 with Maven but without a pom.xml, just the artifacts
+  runtimeOnly group: 'org.sosy-lab', name: 'javasmt-solver-z3', version: '4.8.10', classifier: 'com.microsoft.z3', ext: 'jar'
```

Comment:
Is there a way to use a central variable for the version numbers of SMT solvers?
Maybe something like `z3.version=4.8.10`.

Reply:
Yes, i've added that in the latest commit.